dotCMS/src/main/webapp/html/js/dotcms/dijit/FileBrowserDialog.jsp

```diff
@@ -61,7 +61,7 @@
                                 </tbody>
                             </table>
                         </div>
-                        <div dojoAttachPoint="tablesSummary" style="display:none" class="portlet-pagination">
+                        <div dojoAttachPoint="tablesSummary" style="display:none" class="file-browser portlet-pagination">
```

Comment:
There is already a "block" here: ```.file-selector-tree``` and you can see that there is ```.file-selector-tree__toolbar``` and ```file-selector-tree__table``` so for this you should create ```file-selector-tree__pagination``` instead of adding ```.portlet-pagination```

**DON'T remove the one that was already there because we don't know who can be using it**